src/server/engine_shard_set.cc

```diff
@@ -30,6 +32,17 @@ ABSL_FLAG(bool, cache_mode, false,
           "If true, the backend behaves like a cache, "
           "by evicting entries when getting close to maxmemory limit");
 
+// memory defragmented related flags
```

Comment:
I would't understand what those flags mean If I didn't look at the code. Probably not important now, but should be maybe a to-do. 

"Minimum percentage of used memory relative to total available memory before running defragmentation"

Reply:
Yes I didn't like this one as well :).